guides/common/modules/proc_configuring-host-based-authentication-control.adoc

```diff
@@ -1,62 +1,111 @@
 
-HBAC rules define which machine within the domain a {FreeIPA} user is allowed to access.
-You can configure HBAC on the {FreeIPA} server to prevent selected users from accessing {ProjectServer}.
-With this approach, you can prevent {Project} from creating database entries for users that are not allowed to log in.
+In {FreeIPA}, host-based access control (HBAC) rules define which {FreeIPA} users can access which {FreeIPA} hosts and which {FreeIPA} services can be used to gain access.
+For example, you can configure HBAC on the {FreeIPA} server to limit access to {ProjectServer} only to selected users or user groups.
+By configuring a HBAC rule in the {FreeIPA} domain, you can ensure {Project} does not create database entries for users who should not have access.
+
+.Prerequisites
+
+* {FreeIPA} user account with privileges to configure HBAC rules
+
+.Procedure
+
+. On the {FreeIPA} server, configure HBAC control. For more information, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
+.. Create a HBAC service for {ProjectServer}.
+.. Create a new HBAC rule to define the required access control.
+Add the following {FreeIPA} entities to the HBAC rule:
+... The HBAC service for {ProjectServer}
+... The {ProjectServer} host
+... The users or user groups to whom you want to grant access
+.. Make sure the default {FreeIPA} `allow_all` rule is disabled.
 ifndef::orcharhino[]
-For more information on HBAC, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
+For information about how to disable `allow_all` without disrupting other services, see the https://access.redhat.com/solutions/67895[How to configure HBAC rules in IdM] article on the Red{nbsp}Hat Customer Portal.
 endif::[]
+. On your {ProjectServer}, load the host-based access control rules from {FreeIPA}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-pam-service=satellite-prod
+----
 
-On the {FreeIPA} server, configure Host-Based Authentication Control (HBAC).
+.Verification
 
-.Procedure
-. On the {FreeIPA} server, to authenticate, enter the following command and enter your password when prompted:
+In {ProjectWebUI}:
+
+. Log in as a user defined in {FreeIPA} who is included in the HBAC rule.
```

Comment:
Maybe as a line:
. Log in to the {ProjectWebUI} as a user defined in ...
+
If the user is not included in the HBAC rule, access should be denied.